clippy_lints/src/utils/sugg.rs

```diff
@@ -312,7 +312,7 @@ pub fn make_assoc(op: AssocOp, lhs: &Sugg, rhs: &Sugg) -> Sugg<'static> {
         AssocOp::AssignOp(op) => format!("{} {}= {}", lhs, token_to_string(&token::BinOp(op)), rhs),
         AssocOp::As => format!("{} as {}", lhs, rhs),
         AssocOp::DotDot => format!("{}..{}", lhs, rhs),
-        AssocOp::DotDotDot => format!("{}...{}", lhs, rhs),
+        AssocOp::DotDotEq => format!("{}...{}", lhs, rhs),
```

Comment:
Should this be changed to `..=`?

Reply:
yeah, that's why it's not landed yet, i need to do this

Reply:
(bit busy atm)